### ISSUE TITLE
The nvidia-dcgm-exporter does not show GPU metrics.

### ISSUE THREAD

**Issue by avolkov1** [NVIDIA/deepops#773]
The dcgm exporter is being started without `--gpus all` flag. Either use `--gpus all` or `--runtime=nvidia`. Because of this the GPU metrics are not displayed in graphana

Please fix:
https://github.com/NVIDIA/deepops/blob/e76b4a32f4744c46aac62d296716f33e8ffbf3e1/roles/nvidia-dcgm-exporter/templates/docker.dcgm-exporter.service.j2#L12

Should be:
```
ExecStart=/usr/bin/docker run --gpus all --rm --name %n -p 9400:9400 {{ nvidia_dcgm_container }}
```

This might be an issue only when the default runtime is not set to nvidia i.e. in file `/etc/docker/daemon.json` when "default-runtime" is not set.
```
{
    "default-runtime": "nvidia",
    "runtimes": {
        "nvidia": {
            "path": "/usr/bin/nvidia-container-runtime",
            "runtimeArgs": []
        }
    }
}
```

I encountered this issue when deploying only SLURM (and not the Kubernetes components).

**Comment by supertetelman** [comment]
For a while we were assigning nvidia as the default runtime. It's possible that this behavior was changed and we need to fix this. Doug, can you take a look?

**Comment by dholt** [comment]
We just don't have a method to set the default runtime any longer. Using Alex's suggestion for now.

**Comment by avolkov1** [comment]
Fixed in PR #850.
